src/it/resources/data/typicalDataBundle.json

```diff
@@ -489,38 +461,15 @@
     "comment1ToResponse1ForQ1": {
       "feedbackResponse": {
         "id": "00000000-0000-4000-8000-000000000901",
-        "feedbackQuestion":
-        {
+        "feedbackQuestion": {
           "id": "00000000-0000-4000-8000-000000000801",
           "feedbackSession": {
             "id": "00000000-0000-4000-8000-000000000701"
           },
           "questionDetails": {
             "questionType": "TEXT",
             "questionText": "What is the best selling point of your product?"
```

Comment:
Actually these are redundant as well, possible to remove or too much hassle?

Reply:
Updated!